caproto/_broadcaster.py

```diff
@@ -97,16 +114,27 @@ def recv(self, byteslike, address):
         -------
         commands : list
         """
-        self.log.debug("Received datagram from %s:%d with %d bytes.",
-                       *address, len(byteslike))
         try:
             commands = read_datagram(byteslike, address, self.their_role)
         except Exception as ex:
             raise RemoteProtocolError(f'Broadcaster malformed packet received:'
                                       f' {ex.__class__.__name__} {ex}') from ex
 
+
+        tags = {'their_address': address,
+            'direction': '<<<---',
+            'role': repr(self.our_role)}
         for command in commands:
-            self.log.debug("%d bytes -> %r", len(command), command)
+            if isinstance(command, Beacon):
+                self.beacon_log.debug("%s:%d (%dB) %r", *address, len(command), command, extra=tags)
+            elif hasattr(command, 'ip'):
```

Comment:
It looks like there is no difference between the code in this `elif` block and the code in the `else` block. Would it be equivalent to just remove the `elif` block?